README.md

```diff
@@ -196,21 +196,21 @@ You may be thinking:
 + Why use this approach when CSS has been written
 like the bootstrap example for years?
 + Isn't this just like using inline styles? ([considered by many to be bad practice in CSS](http://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css))
-+ Won't I end up _repeating myself_ in the html?
++ Won't I end up _repeating myself_ in the HTML?
```

Comment:
_technically_ HTML is an initialisation so having the letters in Caps makes sense. 
on W3C HTML is _always_ written in caps: 
https://www.w3.org/standards/webdesign/htmlcss